### PR TITLE
Fix typo in the-wharf-ci-yml-file.md

### PR DESCRIPTION
**Pull request by Pikabanga**

### CODE REVIEW

File: docs/usage-wharfyml/the-wharf-ci-yml-file.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 <!-- div:left-panel -->
 
 The `.wharf-ci.yml` file is located at the **root** path of each projects
-repositories. The file is sometimes referred to as the projects
+repositories. The file is sometimes referred to as the project's
```
Comment (jilleJr) [inline docs/usage-wharfyml/the-wharf-ci-yml-file.md:8]: I always mess this up. Doesn't "project's" mean "project is"? I've never understood this

**Review by jilleJr — APPROVED** [review]
Got a question but I have faith in your English knowledge

File: docs/usage-wharfyml/the-wharf-ci-yml-file.md (hunk 1 above)
Comment (Pikabanga) [reply docs/usage-wharfyml/the-wharf-ci-yml-file.md:8]: Nope, it describes posession :) 
https://en.wikipedia.org/wiki/English_possessive#Nouns_and_noun_phrases

Comment (Pikabanga) [reply docs/usage-wharfyml/the-wharf-ci-yml-file.md]: I just pushed a few more grammar fixes

Comment (jilleJr) [reply docs/usage-wharfyml/the-wharf-ci-yml-file.md]: > The possessive /.../ is made by suffixing a morpheme which is represented orthographically as 's

*Uuuuuh sure*

Jokes aside, that wiki page explained so much I didn't grasp. Thanks for that! Never knew about the "Formation of possessive construction" concept before.

**Review by jilleJr — APPROVED** [review]

**Review by iverestefans — APPROVED** [review]
LGTM!

**Pull request merged**